Android/Google.Android.DataTransport/build.cake

```diff
@@ -7,21 +7,19 @@
 
 var TARGET = Argument ("t", Argument ("target", "ci"));
 
-string v="2.2.0";
-
 Dictionary<string, string> URLS_ARTIFACT_FILES= new Dictionary<string, string>()
 {
 	{
-		$"https://maven.google.com//com/google/android/datatransport/transport-api/{v}/transport-api-{v}.aar",
-		$"./externals/android/transport-api-{v}.aar"
+		$"https://maven.google.com//com/google/android/datatransport/transport-api/2.2.0/transport-api-2.2.0.aar",
+		$"./externals/android/transport-api-2.2.0.aar"
```

Comment:
All the other versions are `2.2.1`, should this one be `2.2.0`?

Reply:
Yes. That was the reason for those 2 additional PRs. Google begun publishing those libraries out of sync.
Now I have to change tagging (tag naming) too. It was version based and now I have to do date based to have it in porper ordeer.